v2f/src/test/scala/org/broadinstitute/monster/etl/v2f/tsvTestFileMissingValues.txt

```diff
@@ -0,0 +1,3 @@
+key1	key2	key3	key4	key5	key6
+v11	v21		v41	v51	
+v12	v22	v32		v52	
```

Comment:
To cover the `trim` bug I mentioned above, can you take one of the empty cells here and replace it with a whitespace character?